src/test/util/setup_common.h

```diff
@@ -286,6 +286,12 @@ inline std::ostream& operator<<(std::ostream& os, const std::optional<T>& v)
     return v ? os << *v
              : os << "std::nullopt";
 }
+
+template <typename Clock, typename Duration>
+inline std::ostream& operator<<(std::ostream& os, const std::chrono::time_point<Clock, Duration>& tp)
```

Comment:
Where is this being used ?

Reply:
If you remove it and try to compile, you'll notice that Boost requires these for the test to be able to print out the the values when there's a mismatch, see: https://github.com/bitcoin/bitcoin/blob/fae29c5dd71b263fb3bda8d554c9e6a4908175db/src/test/testnet4_miner_tests.cpp#L49